auraescript/lib/temp/runtime.ts

```diff
@@ -0,0 +1,431 @@
+/* eslint-disable */
```

Comment:
Did you want `temp` to be in the PR?

Reply:
There is definitely a cleaner organization possible so that the temp dir won't be needed either, but I wanted to get the fix in ASAP

Reply:
Looks like I figured it out. Will try to eliminate the intermediary files (temp) being output in a future refactor. Probably after the holiday.